src/libfuncs/felt252.rs

```diff
@@ -217,28 +218,23 @@ pub fn build_binary_operation<'ctx, 'this>(
             // Egcd works by calculating a series of remainders, each the remainder of dividing the previous two
             // For the initial setup, r0 = PRIME, r1 = a
             // This order is chosen because if we reverse them, then the first iteration will just swap them
-            let prev_remainder = start_block
-                .append_operation(arith::constant(context, attr_prime_i512, location))
-                .result(0)?
-                .into();
+            let prev_remainder = start_block.append_op_result(arith::constant(
+                context,
+                attr_prime_i512,
+                location,
+            ))?;
             let remainder = start_block.argument(0)?.into();
             // Similarly we'll calculate another series which starts 0,1,... and from which we will retrieve the modular inverse of a
-            let prev_inverse = start_block
-                .append_operation(arith::constant(
-                    context,
-                    IntegerAttribute::new(i512, 0).into(),
-                    location,
-                ))
-                .result(0)?
-                .into();
-            let inverse = start_block
-                .append_operation(arith::constant(
-                    context,
-                    IntegerAttribute::new(i512, 1).into(),
-                    location,
-                ))
-                .result(0)?
-                .into();
+            let prev_inverse = start_block.append_op_result(arith::constant(
+                context,
+                IntegerAttribute::new(i512, 0).into(),
+                location,
+            ))?;
+            let inverse = start_block.append_op_result(arith::constant(
+                context,
+                IntegerAttribute::new(i512, 1).into(),
+                location,
+            ))?;
```

Comment:
Here I think you can just use the block ext `const_int` method no?

Reply:
Hi, is there an update on this? Can we try to get it merged? 😁

Reply:
If you're aiming for merging already you should remove the "[DNM] " prefix from the title, as it literally means "do not merge", implying you don't consider it ready.